### PR TITLE
make getElementName() method variadic & add phpunit test for it

### PR DESCRIPTION
**Pull request by aktorou**

### CODE REVIEW

File: src/BemComponentTrait.php [hunk 1]
```diff
@@ -3,19 +3,39 @@
...
   abstract public function getBlockName(): string;
 
-  public function getModifier(string $modifier, string $element = null): string
+  /**
+   * @param string $modifier
+   * @param string ...$elements
+   *
+   * @return string
+   */
+  public function getModifier(string $modifier, string... $elements): string
```
Comment (bajb) [inline src/BemComponentTrait.php]:
```suggestion
  public function getModifier(string $modifier, string ...$elements): string
```

File: src/BemComponentTrait.php [hunk 2]
```diff
@@ -3,19 +3,39 @@
...
     return $this->getBlockName() . '--' . $modifier;
   }
 
-  public function getElementName(string $element): string
+  /**
+   * @param string ...$elements
+   *
+   * @return string
+   */
+  public function getElementName(string... $elements): string
```
Comment (bajb) [inline src/BemComponentTrait.php]:
```suggestion
  public function getElementName(string ...$elements): string
```

**Review by bajb — CHANGES_REQUESTED** [review]

File: src/BemComponent.php [hunk 3]
```diff
@@ -3,5 +3,6 @@
 
 interface BemComponent
 {
+  /** @return string */
```
Comment (bajb) [inline src/BemComponent.php]: If you are adding a docbloc, please also put a description.

The docbloc is wasted when we have the return type already

Comment (aktorou) [reply src/BemComponent.php]: have added a short description to each

**Review by bajb — APPROVED** [review]

**Pull request merged**